app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,78 @@
+class MoviesController < ApplicationController
+  before_action :set_movie, only: %i[show edit update destroy]
+
+  # GET /movies or /movies.json
+  def index
+    begin
+      @movies = Contexts::Movies::Queries::MovieQueries.new.all
+    rescue ActiveRecord::CatchAll
+      render json: 'records not found'
+    end
+    render json: @movies
+  end
+
+  # GET /movies/1 or /movies/1.json
+  def show; end
+
+  # GET /movies/new
+  def new
+    @movie = Movie.new
+  end
+
+  # GET /movies/1/edit
+  def edit; end
+
+  # POST /movies or /movies.json
+  def create
+    respond_to do |format|
+      @movie = Contexts::Movies::Commands::Create.new.call(params: movie_params)
+      if @movie.errors.size > 0
+        format.html { render :new, status: :unprocessable_entity }
+        format.json { render json: @movie.errors, status: :unprocessable_entity }
+      else
+        format.html { redirect_to movie_url(@movie), notice: 'Movie was successfully created.' }
+        format.json { render :show, status: :created, location: @movie }
+      end
+    end
+  end
+
+  # PATCH/PUT /movies/1 or /movies/1.json
+  def update
+    respond_to do |format|
+      @movie = Contexts::Movies::Commands::Update.new.call(params: movie_params)
+      if @movie.errors.size > 0
+        format.html { render :edit, status: :unprocessable_entity }
+        format.json { render json: @movie.errors, status: :unprocessable_entity }
+      else
+        format.html { redirect_to movie_url(@movie), notice: 'Movie was successfully updated.' }
+        format.json { render :show, status: :ok, location: @movie }
+      end
+    end
```

Comment:
okay, so for now let's consider case when `find` will rise `ActiveRecord::RecordNotFound` exception

```suggestion
    end
    rescue ActiveRecord::RecordNotFound
      format.html { render :edit, status: :unprocessable_entity }
      format.json { render json: @movie.errors, status: :unprocessable_entity }
```
i don't really like such handling, but be can change it later in other tasks,
ok for now